sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/Customization/NodeInput.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Media.VideoAnalyzer.Edge.Models
+{
+    /// <summary>
+    /// Extension for <see cref="NodeInput"/>.
```

Comment:
These aren't extension methods. I'd leave off the comment if the swagger-provided one the generator used is good enough, or write something more descriptive for users who may not be as familiar with this SDK.